com.tuxpusher.TuxPusher.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: com.tuxpusher.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+rename-desktop-file: tuxpusher.desktop
+rename-appdata-file: tuxpusher.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: tuxpusher
+    buildsystem: simple
+    subdir: TuxPusher_SDL
+    build-commands:
+      - make;make DESTDIR=${FLATPAK_DEST} install
```

Comment:
just to be clear, in usage, `DESTDIR` and `PREFIX` are two different things.

`DESTDIR` is for staging, like when building debian package. It is usually prefixed to the `PREFIX`
`PREFIX` is the install location (which `FLATPAK_DEST` is).